admin/config-ui/fields/class-field-google-search-console-intro.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * @package WPSEO\Admin\ConfigurationUI
+ */
+
+/**
+ * Class WPSEO_Config_Field_Social_Profiles_Intro
+ */
+class WPSEO_Config_Field_Google_Search_Console_Intro extends WPSEO_Config_Field {
+
+	/**
+	 * WPSEO_Config_Field_Social_Profiles_Intro constructor.
+	 */
+	public function __construct() {
+		parent::__construct( 'googleSearchConsoleIntro', 'HTML' );
+
+		/* translators: %1$s is the plugin name */
+		$html = sprintf(
+			__( '%1$s integrates with Google Search Console. Search Console is a
```

Comment:
We might consider `sprintf` the value `Google`

Reply:
This isn't done anywhere else in the plugin and doing so partially seems confusing to both translators and developers.